File: Python/card.py
from payment import Payment

class Card(Payment):
  cardNumber = int
  cardDate = str
  cvv = str

  def __init__(self, id, cardNumber, cardDate, cvv):
    super().__init__(id)
    self.cardNumber = cardNumber
    self.cardDate = cardDate
    self.cvv = cvv

File: Java/Card.java
package Java;

public class Card extends Payment {
  Integer cardNumber;
  String cardDate;
  String cvv;

  public Card(Integer id, Integer cardNumber, String cardDate, String cvv) {
    super(id);
    this.cardNumber = cardNumber;
    this.cardDate = cardDate;
    this.cvv = cvv;
  }
}
